README.md

```diff
@@ -16,25 +16,87 @@
 [![CodeScene Code Health](https://codescene.io/projects/9260/status-badges/code-health)](https://codescene.io/projects/9260)
 [![CodeScene System Mastery](https://codescene.io/projects/9260/status-badges/system-mastery)](https://codescene.io/projects/9260)
 
-09/09/2020
+`Initial Release`: 09/09/2020
+
+## Setup
+```shell
+git clone https://github.com/Skiller9090/Lucifer.git
+cd Lucifer
+pip install -r requirements.txt
+python main.py --help
+```
+If you want the cutting edge changes add `-d dev` to the end of `git clone https://github.com/Skiller9090/Lucifer.git`
 
 ## Commands
+| Command            | Description                                                             |
+| ------------------ | ----------------------------------------------------------------------- |
+|help                | Displays This Menu                                                      |
+|name                | Shows name of current shell                                             |
+|id                  | Displays current shell's id                                             |
+|show                | Shows options or modules based on input, EX: show <options/modules>     | 
+|options             | Shows a list of variable/options already set                            |
+|set                 | Sets a variable or option, EX: set <var> <data>                         |
+|set_vars            | Auto sets need variables for loaded module                              |
+|description         | Displays description of the module loaded                               |
+|auto_vars           | Displays is auto_vars is True or False for current shell                |  
```

Comment:
Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/manual/alexspam9090/Lucifer/pullRequest?prid=6396871)